metrics/influxdb.py

```diff
@@ -136,7 +142,7 @@ def push(self, points, database):
 
                 try:
                     conn = httplib.HTTPConnection(self.host_port)
```

Comment:
the signature here is `httplib.HTTPConnection(self, host, port=None)` -- what is going on here?